packages/ui/app/src/api-page/ApiPackageContents.tsx

```diff
@@ -1,49 +1,50 @@
 import * as FernRegistryApiRead from "@fern-fern/registry-browser/api/resources/api/resources/v1/resources/read";
 import { useApiDefinitionContext } from "../api-context/useApiDefinitionContext";
-import { SeparatedElements } from "../commons/SeparatedElements";
 import { joinUrlSlugs } from "../docs-context/joinUrlSlugs";
 import { Endpoint } from "./endpoints/Endpoint";
-import { SectionSeparator } from "./section-separator/SectionSeparator";
 import { ApiSubpackage } from "./subpackages/ApiSubpackage";
 import { doesSubpackageHaveEndpointsRecursive } from "./subpackages/doesSubpackageHaveEndpointsRecursive";
 
 export declare namespace ApiPackageContents {
     export interface Props {
         package: FernRegistryApiRead.ApiDefinitionPackage;
         slug: string;
+        isLastInParentPackage: boolean;
     }
 }
 
-export const ApiPackageContents: React.FC<ApiPackageContents.Props> = ({ package: package_, slug }) => {
+export const ApiPackageContents: React.FC<ApiPackageContents.Props> = ({
+    package: package_,
+    slug,
+    isLastInParentPackage,
+}) => {
     const { resolveSubpackageById } = useApiDefinitionContext();
 
     return (
         <>
-            <SeparatedElements separator={<SectionSeparator />}>
-                {[
-                    ...package_.endpoints.map((endpoint) => (
-                        <Endpoint
-                            key={endpoint.id}
-                            endpoint={endpoint}
-                            slug={joinUrlSlugs(slug, endpoint.urlSlug)}
-                            package={package_}
-                        />
-                    )),
-                    ...package_.subpackages.map((subpackageId) => {
-                        if (!doesSubpackageHaveEndpointsRecursive(subpackageId, resolveSubpackageById)) {
-                            return null;
-                        }
-                        const subpackage = resolveSubpackageById(subpackageId);
-                        return (
-                            <ApiSubpackage
-                                key={subpackageId}
-                                subpackageId={subpackageId}
-                                slug={joinUrlSlugs(slug, subpackage.urlSlug)}
-                            />
-                        );
-                    }),
-                ]}
-            </SeparatedElements>
+            {package_.endpoints.map((endpoint, idx) => (
+                <Endpoint
+                    key={endpoint.id}
+                    endpoint={endpoint}
+                    isLastInApi={isLastInParentPackage && idx === package_.endpoints.length - 1}
```

Comment:
this is def a bit less elegant than what we had before & seems a little more error-prone (eg as we add more api components). what was the issue with `SectionSeparator`?

Reply:
Yes, agreed. Definitely less elegant. If you check [this page](https://vellum-staging.docs.dev.buildwithfern.com/api-reference/generate-stream) it should give you an idea. When you select an error you need to be able to unselect it to get back to the success response. Our idea was that the user should be able to unselect it by clicking anywhere outside of the errors section (empty spaces only). `SectionSeparator` reduces that area and it's not straightforward/elegant to attach an `onClick` event handler to it. So ideally we want to replace it with bottom padding and border.

But I just did a quick research and it looks like we can just use Tailwind's [`divide-width`](https://tailwindcss.com/docs/divide-width#add-borders-between-stacked-children) utility for this. May be ideal, wdyt?